source/assets/js/components/fef-swipeable-area.js

```diff
@@ -101,34 +113,45 @@ export class FefSwipeableArea {
             this.$buttonBack.on('click', () => { this.pageBack(); });
             this.$buttonForward.on('click', () => { this.pageForward(); });
 
-            // hinting = showing a little bit of the remaining elements on hovering over the buttons.
-            // Can be supplied in pixels via data-attribute, e.g. data-swipeable-hinting="20"
-            const hintAmount = parseInt(this.$element.data('swipeable-hinting'));
+            this.setupHinting();
+        }
+    }
 
-            if (hintAmount > 0) {
-                this.$buttonBack.hover(this.hintOn(this.$buttonBack, hintAmount), this.hintOff(this.$buttonBack));
-                this.$buttonForward.hover(this.hintOn(this.$buttonForward, -hintAmount), this.hintOff(this.$buttonForward));
-            }
+    /**
+     * When hovering over a button (forward or backward), the container content
+     * will be "hinted at", i.e. moved into view a bit more.
+     */
+    setupHinting() {
+        // hinting = showing a little bit of the remaining elements on hovering over the buttons.
+        const useHinting = !!this.$element.data('swipeable-hinting');
+
+        if (useHinting) {
+            this.$buttonBack
+                .addClass(BUTTON_HINTABLE_CLASS)
+                .hover(
+                    () => this.applyHint(HINT_AMOUNT),
+                    () => this.applyHint(0)
+                );
+            this.$buttonForward
+                .addClass(BUTTON_HINTABLE_CLASS)
+                .hover(
+                    () => this.applyHint(-HINT_AMOUNT),
+                    () => this.applyHint(0)
+                );
         }
     }
 
+    /**
+     * Shows/hides the buttons via class if needed.
+     * - Buttons only appear on Breakpoints Desktop and Desktop Wide
+     * - If scrolled to the very end, don't show the forward button
+     * - If scrolled to the very beginning, don't show the back button
+     */
     updateButtonStatus() {
-        if (FefResponsiveHelper.isDesktop() || FefResponsiveHelper.isDesktopWide()) {
-            // show forward button if needed
-            if (this.isAtScrollEnd()) {
-                this.$buttonForward.removeClass(BUTTON_ACTIVE_CLASS);
-            } else if (this.hasScrollableOverflow()) {
-                this.$buttonForward.addClass(BUTTON_ACTIVE_CLASS);
-            } else {
-                this.$buttonForward.removeClass(BUTTON_ACTIVE_CLASS);
-            }
-
-            // show back button if needed
-            if (this.hasScrollableOverflow() && this.$innerContainer.scrollLeft() > BUTTON_BACK_THRESHOLD) {
-                this.$buttonBack.addClass(BUTTON_ACTIVE_CLASS);
-            } else {
-                this.$buttonBack.removeClass(BUTTON_ACTIVE_CLASS);
-            }
+        // show forward/back buttons if needed
+        if (this.hasScrollableOverflow() && FefResponsiveHelper.isDesktopUp) {
```

Comment:
Sollte ein Methodenaufruf sein - die "()" in FefResponsiveHelper.isDesktopUp fehlen. Hat unten noch weitere solche Fälle.